analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/Binary.Arithmetic.cs

```diff
@@ -138,4 +138,20 @@ private static NumberConstraint CalculateAnd(NumberConstraint left, NumberConstr
             return BigInteger.Max(left.Max.Value, right.Max.Value);
         }
     }
+
+    private static BigInteger? BiggestMinimum(NumberConstraint left, NumberConstraint right)
+    {
+        if (left.Min is null)
+        {
+            return right?.Min;
+        }
+        else if (right?.Min is null)
+        {
+            return left.Min;
+        }
+        else
+        {
+            return BigInteger.Max(left.Min.Value, right.Min.Value);
+        }
+    }
```

Comment:
Make it null-safe for both parameters. Same for `SmallestMaximum`.
For consistency, I'd also make `BiggestMinimum` null-safe (returning `null` if at least one parameter is null).

Reply:
That would not be coverable, because for now, all invocations are guaranteed to already have NotNull at least on the left side. We can change it whenever we need.